react-app/src/account/Account.js

```diff
@@ -0,0 +1,86 @@
+import scrypt from 'scrypt-async';
+import nacl from 'tweetnacl';
+import sha256 from 'fast-sha256';
+import blake from 'blakejs';
+
+nacl.util = require('tweetnacl-util');
+
+class Account {
+  constructor(username, password) {
```

Comment:
A constructor must not do async work.

References:
https://stackoverflow.com/a/24686979/687989
https://bytearcher.com/articles/asynchronous-call-in-constructor/

We should have an `init` method that does the keys creation.
Since the key creation is async, we will be able to "hook" to the `init()` method and be sure that the account is ready to be used, right now we can't know since the notification callback is internal to the constructor.

Reply:
Yes, I realized this when writing the constructor, but if I didn't misunderstand, scrypt() is syncronous if interruptStep parameter is undefined or set to 0, isn't it? [scrypt-async repo](https://github.com/dchest/scrypt-async-js)

Reply:
Done